census/estimator/hptuning_config.yaml

```diff
@@ -0,0 +1,22 @@
+trainingInput:
+    hyperparameters:
+        goal: MAXIMIZE
+        hyperparameterMetricTag: accuracy
+        maxTrials: 10
+        maxParallelTrials: 2
+        params:
+          - parameterName: first-layer-size
+            type: INTEGER
+            minValue: 50
+            maxValue: 500
+            scaleType: UNIT_LINEAR_SCALE
+          - parameterName: num-layers
+            type: INTEGER
+            minValue: 1
+            maxValue: 15
+            scaleType: UNIT_LINEAR_SCALE
+          - parameterName: scale-factor
+            type: DOUBLE
+            minValue: 0.1
```

Comment:
No it won't. I lower bounded layers to size 2 in the code as @brandondutra suggested.